src/d3a/models/area/stats.py

```diff
@@ -30,6 +32,24 @@ def __init__(self, area_markets):
         self.aggregated_stats = {}
         self.market_bills = {}
         self.rate_stats_market = {}
+        self.exported_energy = {}
+        self.imported_energy = {}
+
+    def aggregate_exported_imported_energy(self, area):
```

Comment:
I would strongly suggest to decouple it to its own class, that contains only `exported_energy` and `imported_energy` as members. That way the separation is going to be clearer, and you will not have to construct the AreaStats class twice just to use this function.

Reply:
@spyrostz Why do I have to construct the AreaStats class twice? 
I dont see it

Reply:
Ah now I see it better, you do not really reconstruct the AreaStats object, but you simply store the `imported_energy` `exported_energy` parameters and the `aggregate_exported_imported_energy` here. If it is only used by the `BaselinePeakEnergyStats` class, why not store these parameters and methods there ? It is a bit misleading that these exist in a different class / module. 
Of course, this is under the assumption that these stats are not going to be visible via the external connection. If they indeed need to be exposed via the external connection it is ok to have them here. But I think that we are not going to publish them if I understand correctly.